frontend/src/components/SearchBox.js

```diff
@@ -27,6 +26,29 @@ const SearchBox = () => {
     pushKeyword(keyword);
   };
 
+// defining the function of the shortcut for the searh bar
+const ShortcutKey = (key, callback)=> {
+  const callbackRef = useRef(callback)
+  useEffect(() => {callbackRef.current = callback})
+
+  useEffect(() => {
+    function handle(event){
+      if (event.code === key) callbackRef.current(event)
+    }
+      document.addEventListener('keyup', handle)
+      return () => document.addEventListener('keyup', handle)
+  }, [key])
+}
+
+// Applying the functionality of the shortcut to the search bar
+ShortcutKey ("Slash",(event)=>{
+  // if event occured in an input or textarea do nothing
+  if (event.target.closest('input, textarea')) return;
+  
+  // else focus on the searchbox
+  else{document.getElementById('search-input').focus()}
```

Comment:
Could you run this through lint?  So 'npm run format'.  I'm seeing some small lint issues